.goreleaser.yml

```diff
@@ -0,0 +1,43 @@
+before:
+  hooks:
+    - go mod download
+builds:
+  - env:
+      - CGO_ENABLED=0
+    id: k8gb 
+    main: ./main.go
+    binary: bin/manager
+    goos:
+      - linux
+    goarch:
+      - amd64
+      - arm64
+dockers:
+- image_templates:
+  - "absaoss/k8gb:{{ .Tag }}-amd64"
+  use_buildx: false
+  dockerfile: Dockerfile-gr-amd64
+  build_flag_templates:
+  - "--platform=linux/amd64"
+- image_templates:
+  - "absaoss/k8gb:{{ .Tag }}-arm64"
+  use_buildx: false
+  goarch: arm64
+  dockerfile: Dockerfile-gr-arm64
+  build_flag_templates:
+  - "--platform=linux/arm64"
+docker_manifests:
+- name_template: absaoss/k8gb:{{ .Tag }}
+  image_templates:
+  - absaoss/k8gb:{{ .Tag }}-amd64
```

Comment:
Will the final tag include `$arch` suffix?

If yes, it will break this logic https://github.com/AbsaOSS/k8gb/blob/master/chart/k8gb/templates/operator.yaml#L29

Reply:
It follows same strategy as of now. We build two images imag-{.Arch} then wrap it with single manifest name_template: absaoss/k8gb:{{ .Tag }}

Reply:
So it won't break ``Deployment`` manifest. absaoss/k8gb:{{ .Tag }} will resolve into absaoss/k8gb:{{ .Tag }}-amd64 if kubernetes worker is amd64

Reply:
Cool!